### PR TITLE
Accessibility testing

### PR DESCRIPTION
**Pull request by davidragsdale**

### CODE REVIEW

File: scripts/run_pa11y.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+const spawn = require('cross-spawn');
```
Comment (davidragsdale) [inline scripts/run_pa11y.js:1]: @tgardnerFusion I'm curious of your thoughts on this script. On the plus side, it automates the pa11y process which normally has to be run manually against a running server. On the negative side, it is a cumbersome script that may scare devs Thoughts?

File: package.json [hunk 2]
```diff
@@ -36,11 +38,11 @@
     "@babel/plugin-proposal-object-rest-spread": "^7.6.2",
     "@babel/preset-env": "^7.6.3",
     "@fullhuman/postcss-purgecss": "^1.1.0",
-    "ajv": "^6.3.0",
     "autoprefixer": "^9.3.1",
     "babel-loader": "^8.0.0",
     "bootstrap": "^4.3.1",
     "clean-webpack-plugin": "^0.1.19",
+    "cross-spawn": "^7.0.2",
```
Comment (tgardnerFusion) [inline package.json:45]: @davidragsdale I don't think you need this anymore now that you revised the approach.

Comment (davidragsdale) [reply package.json:45]:
```suggestion
```

File: ROADMAP.md [hunk 3]
```diff
@@ -60,26 +60,26 @@ Starting from scratch, create standard configuration that can be used with almos
...
+  - Script bundling ~~[#9](https://github.com/fusionalliance/starter-kit/pull/9)~~
+  - CSS pre-processing ~~[#9](https://github.com/fusionalliance/starter-kit/pull/9)~~ ~~[#11](https://github.com/fusionalliance/starter-kit/pull/11)~~
+  - Image optimization [#26](https://github.com/fusionalliance/starter-kit/pull/26)
+  - Minification ~~[#9](https://github.com/fusionalliance/starter-kit/pull/9)~~
   - **Status**: In review. HBS, JS, and SCSS builds complete. Image optimization pending.
 * Quality assurance
-  - Accessibility (e.g. pa11y)
-  - Lint scripts ~~[#12](https://github.com/quicksolutions/starter-kit/pull/12)~~
-  - Commit hooks (e.g. Husky, prettier, eslint fix) ~~[#12](https://github.com/quicksolutions/starter-kit/pull/12)~~
+  - Accessibility (e.g. pa11y) [#27](https://github.com/fusionalliance/starter-kit/pull/27)
```
Comment (tgardnerFusion) [inline ROADMAP.md:78]: @davidragsdale Sorry, missed this. This should be crossed out as a matter of completing this PR.

**Pull request merged**